src/ds/ds-motion-common.cpp

```diff
@@ -76,10 +76,11 @@ namespace librealsense
 
     rs2_intrinsics ds_fisheye_sensor::get_intrinsics(const stream_profile& profile) const
     {
+        // ds5 used because no fisheye in ds6
         auto fisheye_calib = get_fisheye_calibration_table();
-        return get_intrinsic_by_resolution(
+        return get_ds5_intrinsic_by_resolution(
```

Comment:
Here to, if it's `ds5_` it's not common, right?

Reply:
Well, since there is no fisheye sensor in ds6 for now, I used ds5 - see comment at line 79